internal/server/polling/server.go

```diff
@@ -97,9 +100,75 @@ func (s *Server) poll(ctx context.Context, resource types.NamespacedName, secret
 		return fmt.Errorf("failed to list pull requests: %w", err)
 	}
 
+	// Log the PRs.
 	for _, pr := range prs {
 		s.log.Info("pull request", "pr", pr)
 	}
```

Comment:
Don't forget to remove this.

Reply:
Still a draft PR. It's a leftover from your code. Log messages are there for convenience. Removed.